frontend/homework/9/src/App.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import "./App.css";
+import ProfileContainer  from "./containers/Profile/ProfileContainer";
+
+
+function App() {
+  const profileData = {
+    "name": "Amey",
+    "fullName": "Amey Aditya",
+    "qualification": "SSE",
+    "skills": [
+      {
+        "id": 1,
+        "skill": "Python"
+      },
+      {
+        "id": 2,
+        "skill": "React"
+      }
+    ],
+    "hobbies": [
+      {
+        "id": 1,
+        "hobby": "Cricket"
+      }
+    ]
+  };
```

Comment:
Can be abstracted out in a json file OR create and use utils file to expose a function which can return this.